sources/co/vac/cartago.json

```diff
@@ -0,0 +1,34 @@
+{
+    "coverage": {
+        "ISO 3166": {
+            "alpha2": "CO-VAC",
+            "country": "Cartago",
+            "region": "Valle del Cauca"
+        },
+        "country": "co",
+        "state": "vac"
+    },
+    "schema": 2,
+    "layers": {
+        "addresses": [
+            {
+                "name": "city",
+                "data": "https://addressforall-my.sharepoint.com/personal/operacao_addressforall_org/_layouts/15/download.aspx?share=EdyPQRg2TShKgpnV5DD2Y0oBZbdPpsjUcI0qVczflqdwOQ",
+                "website": "https://github.com/digital-guard/preserv-CO/tree/main/data/_pk0006.02",
+                "protocol": "http",
+                "license": {
+                    "text": "CC-BY-4.0R",
```

Comment:
I don't think `CC-BY-4.0R` exists. Is this a typo?

Reply:
I will fix